synapse/cores/common.py

```diff
@@ -257,6 +257,31 @@ def _loadCoreModel(self):
         for item in self.getTufosByProp('syn:prop'):
             self._initPropTufo(item)
 
+    def addDataModel(self, modname, modl):
+        '''
+        Store all types/forms/props from the given data model in the cortex.
+        This should be done on initialization of an empty cortex only.
```

Comment:
Not 100% true... this could be done after the fact to add a new data model to an old cortex.